backend/src/zango/cli/start_project.py

```diff
@@ -228,19 +228,25 @@ def start_project(
     # Creating Public Tenant
     create_public_tenant(platform_domain_url=platform_domain_url)
 
-    # Prompting default platform user details
-    if not platform_username:
-        click.echo("Please enter platform user email")
-        platform_username = click.prompt("Email")
-    if not platform_user_password:
-        platform_user_password = click.prompt(
-            "Password", hide_input=True, confirmation_prompt=True
+     # Prompting default platform user details
+    while True:
+        if not platform_username:
+            click.echo("Please enter platform user email")
+            platform_username = click.prompt("Email")
+        if not platform_user_password:
+            platform_user_password = click.prompt(
+                "Password", hide_input=True, confirmation_prompt=True
+            )
+
+        user_creation_result = create_platform_user(
+            platform_username, platform_user_password
         )
-
-    user_creation_result = create_platform_user(
-        platform_username, platform_user_password
-    )
-    if not user_creation_result["success"]:
-        click.echo("User Creation Failed!")
+        if user_creation_result["success"]:
+            break
+        else:
+            click.echo("User Creation Failed!")
```

Comment:
Instead of a fixed error message, We can display the error message returned in `user_creation_result["message"]`

Reply:
Ahh ok I'll make the edit tonight for both of the files